images/macos/macos-10.15-Readme.md

```diff
@@ -189,21 +190,21 @@
 | 1.18.2  | x64          | GOROOT_1_18_X64      |
 
 ### Rust Tools
-- Cargo 1.60.0
-- Rust 1.60.0
-- Rustdoc 1.60.0
+- Cargo 1.61.0
+- Rust 1.61.0
+- Rustdoc 1.61.0
 - Rustup 1.24.3
 
 #### Packages
 - Bindgen 0.59.2
-- Cargo-audit 0.16.0
+- Cargo-audit 
```

Comment:
Need to fix it.

Reply:
Fixed in https://github.com/actions/virtual-environments/pull/5606